Board.java

```diff
@@ -0,0 +1,634 @@
+package test;
+import java.util.ArrayList;
+
+public class Board {
+    public class BoardSquare {
+        private Tile tile;
+        private boolean isOccupied;
+        private int letterMultiplier;
```

Comment:
what if you need a new multiplier strategy?

Reply:
what do you mean?